index.html

```diff
@@ -55,21 +68,34 @@ <h3>
        <li><a href="#"><img src="./img/Icon-Medium.png" alt="Medium"> </a> </li>
    </ul>
     </div>
-   
+</div>
 </div>
 
 </section>
    
 
 
   <section class="recent-works">
+     
+
+    
     <header class="title-header">
+      
+
+        
         <h2 class="title">My Recent Works</h2>
+    <hr class="title-border">
+
       </header>
+       <div class="d-flex">
       <div class="work">
           <section >
-  <img src="./img/Img-Placeholder.png" alt="">
+  <img src="./img/Img-Placeholder.png" class="mobile-img" alt="">
+  <img src="./img/desk-Img-Placeholder.png" class="desk-img" alt="">
+
+   
```

Comment:
- [ ] [OPTIONAL] To improve the SEO for your next website, you should add a value inside the `alt` attribute. For instance:
```
<img src="image_path" alt="logo-desk">
```